Unix/http/httpclientauth.c

```diff
@@ -2028,17 +2035,15 @@ HttpClient_NextAuthRequest(_In_ struct _HttpClient_SR_SocketData * self, _In_ co
     const gss_OID_desc mechset_avail_elems[] = {
         { 6, "\053\006\001\005\005\002" },                  // Spnego
         { 10, "\x2b\x06\x01\x04\x01\x82\x37\x02\x02\x0a" }, // ntlm
-        { 9, "\052\206\110\206\367\022\001\002\002" },      // mech_krb5
-        { 6, "\053\006\001\005\002\005" }                   // mech_iakerb
+        { 9, "\052\206\110\206\367\022\001\002\002" }      // mech_krb5
```

Comment:
Given that this is now here, can 2029 be deleted? Come to think of it, can 2030 and 2032 be deleted? And 2033/2034?